### ISSUE TITLE
Redesign homepage

### ISSUE THREAD

**Issue by garethrees** [mysociety/whatdotheyknow-theme#212]
Taken over from https://github.com/mysociety/alaveteli/issues/1525

## Metadata

- **Team:** @wrightmartin, @garethrees
- **Prototypes:** http://invis.io/D52MTYYTK
- **Final Mockup:**

![wdtk-home](https://cloud.githubusercontent.com/assets/2292925/7477385/58a95ccc-f34b-11e4-92e8-a2e49f2dcf8f.png)

## Todo

- [x] Improve mySociety Banner (PRs: #194)
- [x] Improve Banner (PRs: #197)
- [x] Improve locale switcher (PRs: #195)
- [x] Improve logged-in nav (Issues: #205, #206), (PRs: #214)
- [x] Improve search box (PRs: https://github.com/mysociety/alaveteli/pull/2273)
- [x] Add hero section (PRs: #198, #210)
- [x] Add how it works section (PRs: #211)
- [x] Improve latest requests listing (PRs: #199)
- [ ] Add user testimonials (Issues: https://github.com/mysociety/alaveteli/issues/2383)
- [x] Improve footer (PRs: #196, #201)
- [x] Improve site-wide fonts (PRs: #215)
- [x] Change primary colour (Red -> Blue) (PRs: #216)
- [x] Tidy up sidebars
- [x] Final polish (Issues: #218, #230, #222) 
- [x] Add note to analytics when launched
- [ ] Refactor `custom.scss` (Issues: #208 ~~https://github.com/mysociety/alavetelitheme/issues/48~~)
- [x] Update `print.scss` (Issues: #223), (PRs: #224)

---

- [x] Blog post / announcement? (Issues: #229)
- [ ] Change other promo material to blue (Issues: #207, #235)
- [ ] Update screenshots on mysociety.org, alaveteli.org, etc (Issues: https://github.com/mysociety/alaveteli/issues/2777)
- [ ] Generate translations (Issues: #234)

**Comment by MyfanwyNixon** [comment]
I've drafted a blog post ready for launch, and would be glad if someone could fact-check it. 
https://blogs.mysociety.org/mysociety/?p=24657&preview=true

* Parts where I particularly need confirmation (because I made assumptions):

> like almost everything we do these days, the changes will be subjected to scrutiny from our Research team.

> As with all site improvements, the design will be rolled out as an option on the wider Alaveteli codebase, so that it can be implemented by anyone running an Alaveteli site.

* Also:

Have we performed user testing? It would be nice to mention that, if so.
Anything else I have missed about the thinking behind the design, @wrightmartin?

Finally it will just need a screenshot dropped in as the feature image, plus it would be nice to include shots of the various page elements as they are mentioned.
